sdcm/nemesis.py

```diff
@@ -1460,11 +1460,9 @@ def generate_random_parameters_values():  # pylint: disable=invalid-name
                                         msg="Wrong number of requested and expected toppartitions for {} sampler".format(
                                             sampler))
 
-    def disrupt_network_random_interruptions(self):  # pylint: disable=invalid-name
-        # pylint: disable=too-many-locals
-        self._set_current_disruption('NetworkRandomInterruption')
-        if not self.cluster.extra_network_interface:
-            raise UnsupportedNemesis("for this nemesis to work, you need to set `extra_network_interface: True`")
+    def get_rate_limit_for_network_disruption(self) -> Optional[str]:
+        if not self.monitoring_set.nodes:
+            return None
```

Comment:
we can make some default but it won't make much sense, 
the idea was to limit between 30%-70% of the current bandwidth, and not do a limit too small or too big that would drop most of the data.